[2023-01-06 02:17:36,375] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: stock_data_pipeline_dag.delete_cluster_dataproc_task manual__2023-01-06T01:21:29.019918+00:00 [queued]>
[2023-01-06 02:17:36,384] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: stock_data_pipeline_dag.delete_cluster_dataproc_task manual__2023-01-06T01:21:29.019918+00:00 [queued]>
[2023-01-06 02:17:36,384] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 02:17:36,384] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-06 02:17:36,384] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 02:17:36,396] {taskinstance.py:1377} INFO - Executing <Task(DataprocDeleteClusterOperator): delete_cluster_dataproc_task> on 2023-01-06 01:21:29.019918+00:00
[2023-01-06 02:17:36,404] {standard_task_runner.py:52} INFO - Started process 11411 to run task
[2023-01-06 02:17:36,409] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'stock_data_pipeline_dag', 'delete_cluster_dataproc_task', 'manual__2023-01-06T01:21:29.019918+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/stock_data_pipeline_dag.py', '--cfg-path', '/tmp/tmp9fzktdrx', '--error-file', '/tmp/tmperr8fskh']
[2023-01-06 02:17:36,410] {standard_task_runner.py:80} INFO - Job 79: Subtask delete_cluster_dataproc_task
[2023-01-06 02:17:36,476] {task_command.py:370} INFO - Running <TaskInstance: stock_data_pipeline_dag.delete_cluster_dataproc_task manual__2023-01-06T01:21:29.019918+00:00 [running]> on host 7aa3417b2d87
[2023-01-06 02:17:36,552] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=stock_data_pipeline_dag
AIRFLOW_CTX_TASK_ID=delete_cluster_dataproc_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-06T01:21:29.019918+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-06T01:21:29.019918+00:00
[2023-01-06 02:17:36,554] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-01-06 02:17:36,554] {dataproc.py:816} INFO - Deleting cluster: move-stock-data
[2023-01-06 02:18:09,071] {dataproc.py:828} INFO - Cluster deleted.
[2023-01-06 02:18:09,088] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=stock_data_pipeline_dag, task_id=delete_cluster_dataproc_task, execution_date=20230106T012129, start_date=20230106T021736, end_date=20230106T021809
[2023-01-06 02:18:09,131] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-06 02:18:09,163] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
